src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -171,77 +188,116 @@ const MovingAverageAggUi = props => {
-            defaultMessage="Settings"
-          />)}
-          helpText={
-            <span>
-              <FormattedMessage
-                id="tsvb.movingAverage.settingsDescription"
-                defaultMessage="{keyValue} space-delimited"
-                values={{ keyValue: (<EuiCode>Key=Value</EuiCode>) }}
-              />
-            </span>
-          }
-        >
-          <EuiTextArea
-            onChange={handleTextChange('settings')}
-            value={model.settings}
-            fullWidth
-          />
-        </EuiFormRow>
-      </EuiFlexItem>
+            <EuiFlexGroup gutterSize="s">
+              {
+                <EuiFlexItem>
+                  <EuiFormRow
+                    id={htmlId('alpha')}
+                    label={i18n.translate('tsvb.movingAverage.alpha', {
+                      defaultMessage: 'Alpha',
+                    })}
+                  >
+                    <EuiFieldNumber
+                      step={0.1}
+                      onChange={handleNumberChange('alpha')}
+                      value={model.alpha}
+                    />
+                  </EuiFormRow>
+                </EuiFlexItem>
+              }
+              {
+                (
+                  model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL_DOUBLE ||
+                  model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL_TRIPLE
+                ) && (
+                  <EuiFlexItem>
+                    <EuiFormRow
+                      id={htmlId('beta')}
+                      label={i18n.translate('tsvb.movingAverage.beta', {
+                        defaultMessage: 'Beta',
+                      })}
+                    >
+                      <EuiFieldNumber
+                        step={0.1}
+                        onChange={handleNumberChange('beta')}
+                        value={model.beta}
+                      />
+                    </EuiFormRow>
+                  </EuiFlexItem>
+                )
+              }
+              {
+                model.model_type === MODEL_TYPES.WEIGHTED_EXPONENTIAL_TRIPLE && (
+                  <Fragment>
+                    <EuiFlexItem>
+                      <EuiFormRow
+                        id={htmlId('gamma')}
+                        label={i18n.translate('tsvb.movingAverage.gamma', {
+                          defaultMessage: 'Gamma',
+                        })}
+                      >
+                        <EuiFieldNumber
+                          step={0.1}
+                          onChange={handleNumberChange('gamma')}
+                          value={model.gamma}
+                        />
+                      </EuiFormRow>
+                    </EuiFlexItem>
+                    <EuiFlexItem>
+                      <EuiFormRow
+                        id={htmlId('period')}
+                        label={i18n.translate('tsvb.movingAverage.period', {
+                          defaultMessage: 'Period (days)',
```

Comment:
I'm not really sure that the period is expressed in `days`. The docs is not actually super clear on that
> Holt-Winters requires a little more handholding than the other moving averages. You need to specify the "periodicity" of your data: e.g. if your data has cyclic trends every 7 days, you would set period = 7. Similarly if there was a monthly trend, you would set it to 30. There is currently no periodicity detection, although that is planned for future enhancements.

Seems speaking about days, but I'd like to investigate this a bit further.

Reply:
The 10 value of the `window` was taken from [the doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-movfn-aggregation.html#_syntax_9) as a default. If we leave an empty string like before we'll got failed request.

![image](https://user-images.githubusercontent.com/16738889/59035130-457ecd80-8875-11e9-8037-8fac075bd977.png)

Yes, we can set a minimal value. Changed to 1.

Reply:
**Days** is removed